src/box/memtx_tx.c

```diff
@@ -2992,7 +3003,7 @@ memtx_tx_snapshot_clarify_slow(struct memtx_tx_snapshot_cleaner *cleaner,
 		assert(entry->from == tuple);
 		tuple = entry->to;
 	}
-
+	memtx_tx_story_gc();
```

Comment:
Snapshot cleaner **must not** collect garbage at least because it may be run from different thread.

Reply:
Sorry, I didn't bear that in mind, removed it.

I added it just for the sake of more often garbage collection.